### PR TITLE
Case 20867: Fix translucent textures not getting reset

### PR DESCRIPTION
**Pull request by SamGondelman**
https://highfidelity.manuscript.com/f/cases/20867/Translucent-textures-are-sometimes-displaying-incorrectly

Test plan:
- See ticket.  Translucent things should render correctly

### CODE REVIEW

File: libraries/model-networking/src/model-networking/ModelCache.cpp [hunk 1]
```diff
@@ -422,10 +422,7 @@ void Geometry::setTextures(const QVariantMap& textureMap) {
 bool Geometry::areTexturesLoaded() const {
     if (!_areTexturesLoaded) {
         for (auto& material : _materials) {
-            // Check if material textures are loaded
-            bool materialMissingTexture = material->isMissingTexture();
-
-            if (materialMissingTexture) {
```
Comment (SamGondelman) [inline libraries/model-networking/src/model-networking/ModelCache.cpp]: the changes in this file have no effect but are cleaner/better

File: libraries/render-utils/src/RenderPipelines.cpp [hunk 2]
```diff
@@ -482,6 +482,7 @@ void RenderPipelines::updateMultiMaterial(graphics::MultiMaterial& multiMaterial
                         auto itr = textureMaps.find(graphics::MaterialKey::ALBEDO_MAP);
                         if (itr != textureMaps.end()) {
                             if (itr->second->isDefined()) {
+                                material->resetOpacityMap();
```
Comment (SamGondelman) [inline libraries/render-utils/src/RenderPipelines.cpp]: this is the fix

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16879/HighFidelity-Beta-PR14801-5be6a31.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16879/HighFidelity-Beta-Interface-PR14801-5be6a31.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16879/HighFidelity-Beta-PR14801-5be6a31.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16879/HighFidelity-Beta-Interface-PR14801-5be6a31.exe

**Comment by qliejahner** [comment]
Fail

Most of the examples in the ticket (such as cop can, sun beams, and tutorial signs inside TheSpot) are displaying correctly, but there is still something wrong with the portals and circular indicators found in the tutorial. As before, only a random selection of these objects display correctly when reloaded.

image

[hifi-log.txt](https://github.com/highfidelity/hifi/files/2809939/hifi-log.txt)

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16899/HighFidelity-Beta-PR14801-5609670.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16899/HighFidelity-Beta-PR14801-5609670.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16899/HighFidelity-Beta-Interface-PR14801-5609670.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16899/HighFidelity-Beta-PR14801-5609670.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16899/HighFidelity-Beta-Interface-PR14801-5609670.exe

**Comment by qliejahner** [comment]
Currently testing

**Comment by qliejahner** [comment]
PASS

All translucent objects from the ticket now consistently appear as expected on multiple machines!

**Pull request merged**